composer.json

```diff
@@ -19,7 +19,8 @@
     "monolog/monolog": "~1.17",
     "paragonie/random_compat": "~1.2",
     "suin/php-rss-writer": "~1.3",
-    "swiftmailer/swiftmailer": "~5.4"
+    "swiftmailer/swiftmailer": "~5.4",
+    "google/apiclient": "1.0.*@beta"
```

Comment:
Alphabetize. Also, why are you using an old version? I see no issue with `"~1.1"` stable.